### PR TITLE
Fix for timeanddate.com

### PR DESCRIPTION
**Pull request by Jckpt**
Fixes: #11954
Before:
image

After:
image

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -23196,6 +23196,21 @@ nav.main .menu-btn .menu-btn-box
...
 ================================
 
+timeanddate.com
+
+INVERT
+.tad-logo
+
+CSS
+#header{
+    background: #232627;
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:23206]: As we can use light and dark theme in Dark Readed it's better to use ${SourceColorToInvert}.

So change `#232627` to `${#DCD9D8}` 😉

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -23196,6 +23196,21 @@ nav.main .menu-btn .menu-btn-box
...
+}
+.analog-clock__hands :not(.analog-clock__hand--seconds):not(.analog-clock__pin-inner){
+    stroke: #E7E0D4
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:23209]: Similar situation as previous one:

Change `#E7E0D4` to `${#181F2B}`

And maybe it's not necessary but semicolon in the end of line looks better than no semicolon. 😉

**Review by Myshor — CHANGES_REQUESTED** [review]
Please make changes.

I could push it by myself to your branch but thanks to them you will learn how it's better to make next fixes. 😉

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Jckpt) [reply src/config/dynamic-theme-fixes.config]: Apologies for the delayed response, and thank you for your input. I've addressed the issues following your instructions. Also, my oversight on the missing semicolon, my bad :D

**Review by Myshor — APPROVED** [review]

**Pull request merged**